readme.md

```diff
@@ -1,61 +1,210 @@
 <br/>
 
 <p align="center">
-  <h1>&nbsp;🤖⚡ &nbsp;<code>Sig</code> - a Solana Zig validator client</h1>
-    <br/>
+  <h1 align="center">&nbsp;🤖⚡ &nbsp;<code>Sig</code> - a Solana Zig validator client</h1>
     <br/>
+<div align="center">
   <a href="https://github.com/syndica/sig/releases/latest"><img alt="Version" src="https://img.shields.io/github/v/release/syndica/sig?include_prereleases&label=version"></a>
   <a href="https://github.com/syndica/sig/actions/workflows/test.yml"><img alt="Build status" src="https://img.shields.io/github/actions/workflow/status/syndica/sig/test.yml?branch=main" /></a>
-  <a href="https://ziglang.org/download"><img alt="Zig" src="https://img.shields.io/badge/zig-master-green.svg"></a>
-  <a href="https://github.com/syndica/sig/blob/main/LICENSE"><img alt="License" src="https://img.shields.io/badge/license-MIT-blue"></a>
+  <a href="https://ziglang.org/download"><img alt="Zig" src="https://img.shields.io/badge/zig-0.11.0-green.svg"></a>
+  <a href="https://github.com/syndica/sig/blob/main/LICENSE"><img alt="License" src="https://img.shields.io/badge/license-GPLv3-blue"></a>
+  </div>
 </p>
 <br/>
 
-## Overview
-
-_Sig_ is a Solana validator client implementation written in Zig.
+_Sig_ is a Solana validator client implemented in Zig. Read the [introductory blog post](https://blog.syndica.io/introducing-sig-by-syndica-an-rps-focused-solana-validator-client-written-in-zig/) for more about the goals of this project.
 <br/>
 <br/>
 
 ⚠️ NOTE: This is a WIP, please open any issues for any bugs/improvements.
 
-## Why Zig?
+## 📋 Setup
+
+Zig 0.11.0 is required to build Sig.
+
+<details><summary>Dev Environment Details</summary>
+
+### Build Dependencies
```

Comment:
should probs move this section out of the drop menu - just have it below setup instead of the sentence